requirements.txt

```diff
@@ -2,7 +2,7 @@
 git+https://github.com/tdryer/hangups.git
 appdirs
 asyncio
-aiohttp
+aiohttp>=1.3,<2 # telepot library also needs this version
```

Comment:
My system has been running on aiohttp 1.3.0 with no issues thus far
I can confirm that aiohttp 1.2.0 raises some errors within the telepot library.